extensions/machine-learning/src/packageManagement/packageManager.ts

```diff
@@ -133,7 +133,10 @@ export class PackageManager {
 
 		await utils.createFolder(utils.getRPackagesFolderPath(this._rootFolder));
 		const packages = this._config.requiredSqlRPackages.filter(p => !p.platform || p.platform === process.platform);
-		await Promise.all(packages.map(x => this.installRPackage(x)));
+		for (let index = 0; index < packages.length; index++) {
```

Comment:
why are we making this sequential? did we run into problems before?

Reply:
because they have dependency to each other. I cannot install them as parallel

Reply:
exactly, It would be nice to add comments about this.

Reply:
sure  I will

Reply:
also, it sounds like the packages need to be added in particular order, please also add comment in the config.json